### PR TITLE
Mention prop_or macro

### PR DESCRIPTION
**Pull request by zoechi**

### CODE REVIEW

**Comment by zoechi** [comment]
Looks like changes were be easier to track if the line length was limited.

**Comment by jstarry** [comment]
I think you mixed up the attributes.

`prop_or_else(..)` takes a function

`prop_or(..)` takes a value

File: src/concepts/components/properties.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Types for which you derive `Properties` must also implement `Clone`. This can be
...
+The fields within a struct that derives `Properties` are required by default. When a field is missing and the component is created in the `html!` macro, a compiler error is returned. For fields with optional properties, use the `#[prop_or_default]` attribute to use the default value for that type when the prop is not specified. To specify a value, use the `#[prop_or(value)]` attribute where value is the default value for the property or alternatively use `#[prop_or_else(function)]` where `function` returns the default value. For example, to default a boolean value as `true`, use the attribute `#[prop_or_else(true)]`. It is common for optional properties to use the `Option` enum which has the default value `None`.
```
Comment (jstarry) [inline src/concepts/components/properties.md]:
```suggestion
The fields within a struct that derives `Properties` are required by default. When a field is missing and the component is created in the `html!` macro, a compiler error is returned. For fields with optional properties, use the `#[prop_or_default]` attribute to use the default value for that type when the prop is not specified. To specify a value, use the `#[prop_or(value)]` attribute where value is the default value for the property or alternatively use `#[prop_or_else(function)]` where `function` returns the default value. For example, to default a boolean value as `true`, use the attribute `#[prop_or(true)]`. It is common for optional properties to use the `Option` enum which has the default value `None`.
```

**Comment by jstarry** [comment]
Thanks @zoechi for fixing this up 👍

**Comment by zoechi** [comment]
Is it ok now? I now just replaced `prop_or` and `prop_or_else`, just before I saw your suggestion.

**Comment by jstarry** [comment]
Yup, it's good!

**Pull request merged**